app/src/main/java/org/mifos/mobile/ui/fragments/RecentTransactionsFragment.kt

```diff
@@ -70,16 +57,22 @@ class RecentTransactionsFragment : BaseFragment(), RecentTransactionsView, OnRef
             inflater: LayoutInflater, container: ViewGroup?,
             savedInstanceState: Bundle?
     ): View? {
-        rootView = inflater.inflate(R.layout.fragment_recent_transactions, container, false)
-        ButterKnife.bind(this, rootView!!)
+         _binding = FragmentRecentTransactionsBinding.inflate(inflater,container,false)
         recentTransactionsPresenter?.attachView(this)
-        sweetUIErrorHandler = SweetUIErrorHandler(activity, rootView)
```

Comment:
We are using "binding.root" twice, it'll be a great for the purpose of clean coding if we create a val rootView = binding.root, and use it wherever Root View is required. Just a suggestion, totally your call.